scripts/subscribeEventProof.js

```diff
@@ -0,0 +1,61 @@
+
+const { Api } = require('@cennznet/api');
+
+async function main (networkName, bridgeContractAddress) {
+    networkName = networkName || 'local';
+    // the API has connected to the node and completed the initialisation process
+    const api = await Api.create({network: networkName});
+
+    // Get the bridge instance that was deployed
+    const Bridge = await ethers.getContractFactory('CENNZnetBridge');
+    console.log('Connecting to CENNZnet bridge contract...');
+    bridgeContractAddress = bridgeContractAddress || "0x5FbDB2315678afecb367f032d93F642f64180aa3";
+    const bridge = await Bridge.attach(bridgeContractAddress);
+    await bridge.deployed();
+    console.log('CENNZnet bridge deployed to:', bridge.address);
+    let eventProofs = [];
+    await api.rpc.chain
+        .subscribeFinalizedHeads(async (head) => {
+           const blockNumber = head.number.toNumber();
+           console.log('At blocknumber:',blockNumber);
+           const blockHash = head.hash.toString();
+           const events = await api.query.system.events.at(blockHash);
+            events.map(async ({event}) => {
+                const { section, method, data } = event;
+                if (section === 'ethBridge' && method === 'AuthoritySetChange') {
+                    const eventIdFound = data.toHuman()[0];
+                    const eventProof = eventProofs.find(ep => ep.eventId === eventIdFound);
+                    console.log('Event proof found::',eventProof);
+                    const { eventId, validatorSetId, signatures } = eventProof;
+                    const signature = ethers.utils.hexlify(signatures[0]);
+                    const sig = ethers.utils.splitSignature(signature);
+                    const cennznetEventProof =
+                            { eventId,
+                              validatorSetId,
+                              v: [sig.v],
+                              r: [sig.r],
+                              s: [sig.s]
+                            };
```

Comment:
should expand this part to work for `n` signatures

Reply:
done